dragonmapper/html.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+"""Formatting Chinese into HTML with dragonmapper's functions"""
+
+
+from __future__ import unicode_literals
+from dragonmapper import hanzi
+from dragonmapper import transcriptions as trans
+from zhon import pinyin
+from zhon import zhuyin
+
+
+"""See recomended CSS style: DRAGONMAPPER_DIR/style.css"""
+
+
+_indentation = 0
+_line_html = ''
+_puctuation = ['，', '。', '“', '”', '：', '；', '？']
```

Comment:
Try `zhon.hanzi.punctuation` and `zhon.pinyin.punctuation`. They are a more exhaustive list.

E.g. `_punctuation = tuple(hanzi.punctuation + pinyin.punctuation)`